ci-operator/config/openshift/cloud-provider-nutanix/OWNERS

```diff
@@ -0,0 +1,8 @@
+approvers:
+- Fedosin
+- JoelSpeed
+- elmiko
+- lobziik
+
+component: "Cloud Compute"
+subcomponent: "Cloud Controller Manager"
```

Comment:
i know this is a CCM, but we have a Nutanix subcomponent now, i wonder if we should use that instead?